rust/streaming_pipe_in.rs

```diff
@@ -0,0 +1,26 @@
+//Script reads text from a named pipe and writes it to stdout, capitalized
+
+use std::env;
+use std::fs::File;
+use std::io::{self, BufRead, BufReader};
+
+fn main() -> io::Result<()> {
```

Comment:
Let's use the `.unwrap()` approach
```suggestion
fn main() {
```